features/cursor-style.yml.dist

```diff
@@ -0,0 +1,126 @@
+# Generated from: cursor-style.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2021-12-07
+  baseline_high_date: 2024-06-07
+  support:
+    chrome: ≤83
+    chrome_android: "83"
+    edge: "83"
+    firefox: ≤72
+    firefox_android: "95"
+    safari: ≤13.1
+    safari_ios: ≤13.4
+compat_features:
+  # baseline: high
+  # baseline_low_date: 2021-12-07
+  # baseline_high_date: 2024-06-07
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "95"
+  #   safari: "1.2"
+  #   safari_ios: "1"
+  - css.properties.cursor
+  - css.properties.cursor.auto
+  - css.properties.cursor.crosshair
+  - css.properties.cursor.default
+  - css.properties.cursor.e-resize
+  - css.properties.cursor.help
+  - css.properties.cursor.inherit
+  - css.properties.cursor.move
+  - css.properties.cursor.n-resize
+  - css.properties.cursor.ne-resize
+  - css.properties.cursor.nw-resize
+  - css.properties.cursor.pointer
+  - css.properties.cursor.progress
+  - css.properties.cursor.s-resize
+  - css.properties.cursor.se-resize
+  - css.properties.cursor.sw-resize
+  - css.properties.cursor.text
+  - css.properties.cursor.w-resize
+  - css.properties.cursor.wait
+
+  # baseline: high
+  # baseline_low_date: 2021-12-07
+  # baseline_high_date: 2024-06-07
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1.5"
+  #   firefox_android: "95"
+  #   safari: "3"
+  #   safari_ios: "1"
+  - css.properties.cursor.alias
+  - css.properties.cursor.all-scroll
+  - css.properties.cursor.cell
+  - css.properties.cursor.col-resize
+  - css.properties.cursor.context-menu
+  - css.properties.cursor.copy
+  - css.properties.cursor.ew-resize
+  - css.properties.cursor.nesw-resize
+  - css.properties.cursor.no-drop
+  - css.properties.cursor.not-allowed
+  - css.properties.cursor.ns-resize
+  - css.properties.cursor.nwse-resize
+  - css.properties.cursor.row-resize
+  - css.properties.cursor.vertical-text
+
+  # baseline: high
+  # baseline_low_date: 2021-12-07
+  # baseline_high_date: 2024-06-07
+  # support:
+  #   chrome: "5"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "3"
+  #   firefox_android: "95"
+  #   safari: "5"
+  #   safari_ios: "4.2"
+  - css.properties.cursor.none
```

Comment:
Up to this point, these are off by one version for Edge when compared to caniuse. I'm not worried about this since it's pre-2020, but I did note it in: https://github.com/web-platform-dx/web-features/issues/1499.